stdlib/public/core/FloatingPoint.swift

```diff
@@ -2042,7 +2042,8 @@ extension FloatingPoint where Self : Differentiable,
   /// result and pullback of `squareRoot` with respect to `self`.
   @inlinable // FIXME(sil-serialize-all)
   func _vjpSquareRoot() -> (Self, (Self) -> Self) {
-    return (squareRoot(), { v in 2 * self * v })
+    let y = squareRoot()
+    return (y, { v in 1 / (2 * y) })
```

Comment:
Wait, `v` is unused?

Reply:
I agree: This should be: v / (2 * y)

Reply:
Good catch folks :) Wrote this at the middle of a night...